pkgs/os-specific/darwin/CoreSymbolication/default.nix

```
@@ -1,19 +1,23 @@
-{ fetchFromGitHub, stdenv }:
+{ lib, fetchFromGitHub, stdenv }:
 
-# Reverse engineered CoreSymbolication to make dtrace buildable
-
-stdenv.mkDerivation rec {
-  name = "CoreSymbolication";
+stdenv.mkDerivation {
+  pname = "core-symbolication";
+  version = "unstable-2018-06-17";
 
   src = fetchFromGitHub {
-    repo = name;
+    repo = "CoreSymbolication";
     owner = "matthewbauer";
     rev = "671fcb66c82eac1827f3f53dc4cc4e9b1b94da0a";
     sha256 = "0qpw46gwgjxiwqqjxksb8yghp2q8dwad6hzaf4zl82xpvk9n5ahj";
```

```suggestion
    rev = "24c87c23664b3ee05dc7a5a87d647ae476a680e4";
    hash = "sha256-PzvLq94eNhP0+rLwGMKcMzxuD6MlrNI7iT/eV0obtSE=";
```

`671fcb6` corresponds to a commit in April 2018. Did you mean `24c87c2` instead?

---

I've split this up into two commits